data/blog/opentelemetry-demo.mdx

```diff
@@ -0,0 +1,272 @@
+---
+title: Exploring OpenTelemetry Demo Application with SigNoz
+slug: exploring-opentelemetry-demo-application-with-signoz
```

Comment:
```suggestion
slug: opentelemetry-demo
```

@Elizabeth-Mathew1 the slug and the filename should be same. The slug is actually controlled by file name. Let's go with opentelemetry-demo as the slug for this.